tcf_website/views/auth.py

```diff
@@ -53,4 +56,4 @@ def logout(request):
     """Logs out user."""
     auth_logout(request)
     messages.add_message(request, messages.SUCCESS, "Logged out successfully!")
-    return redirect('login')
+    return redirect('browse')
```

Comment:
What's the difference between `browse(request)` like the others have and this?

Reply:
I can't use `redirect('browse')` in `login` or `login_error` views without causing a `ERR_TOO_MANY_REDIRECTS`. I'm honestly not sure entirely why